content/docs/command-reference/add.md

```diff
@@ -243,3 +243,39 @@ In this case, a DVC-file is generated for each file in the `pics/` directory
 tree. No top-level DVC-file is generated, which is typically less convenient.
 For example, we cannot use the directory structure as one unit with `dvc run` or
 other commands.
+
+## Example: Dvcignore
+
+To untrack a file or directory just add
+[patterns](https://git-scm.com/docs/gitignore) associated with that file or
+directory under `.dvcignore` file.
```

Comment:
```suggestion
To avoid tracking a file or directory, just add the corresponding
[patterns](https://git-scm.com/docs/gitignore) in a `.dvcignore` file.
```